packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -134,6 +134,15 @@ export class Interpreter {
     while (++this.instructionIndex < this.instructions.length) {
       const instruction = this.instructions[this.instructionIndex];
       if (instruction instanceof Label && instruction.label === label) {
+        // if next immediate op is also label, then keep continuing, otherwise return
+        while (true) {
+          const nextInstruction = this.instructions[this.instructionIndex + 1];
+          if (nextInstruction instanceof Label) {
+            this.instructionIndex++;
+            continue;
+          }
+          break;
```

Comment:
```suggestion
        for (; this.instructionIndex < this.instructions.length-1; ++this.instructionIndex) {
          const nextInstruction = this.instructions[this.instructionIndex + 1];
          if (!(nextInstruction instanceof Label))
            break;
```

Reply:
Thanks for suggestion. Previously for `undefined` case (index out of bound for nextInstruction), `(nextInstruction instanceof Label)` was handling that. 
But this implementation seems more intuitive. Updating